webrender/src/frame.rs

```diff
@@ -612,6 +594,9 @@ impl<'a> FlattenContext<'a> {
                     .clip_rect()
                     .translate(&reference_frame_relative_offset);
                 let content_rect = item.rect().translate(&reference_frame_relative_offset);
+
+                let enable_scrollbars = IframeScrollbars::Enabled == info.enable_scrollbars;
```

Comment:
I turned the enum into a bool here, but I could just as easily thread it all the way down into FrameBuilder if need be.